data/strings/types_strings.txt

```diff
@@ -2011,6 +2011,46 @@
     zh-Hans = 图书馆
     zh-Hant = 圖書館
 
+  [type.amenity.loading_dock]
+    en = Loading dock
```

Comment:
```suggestion
    en = Loading Dock
```
Thanks! Almost great. Please remove regenerate commits, squash all .txt changes into the one commit, and then regenerate strings again.

Reply:
I Now See That Weird Case Dominates, Must Be Some English Thing

But Then

https://github.com/organicmaps/organicmaps/blob/98c3aa20a2b9aa0a31de4d06a4b7a88af5839196/data/strings/types_strings.txt#L51

Likely Needs To Be Changed, Right?

Reply:
We are trying to use Title Case for English since 2011 because it looks better on the Place Page. But looks like not every contributor is aware about it, and we're missing some case issues on reviews.